src/pages/RFOX/components/Stake/StakeInput.tsx

```diff
@@ -101,18 +282,25 @@ export const StakeInput: React.FC<StakeRouteProps> = ({ headerComponent }) => {
               py={4}
               bg='background.surface.raised.accent'
             >
-              <Row fontSize='sm' fontWeight='medium'>
-                <Row.Label>{translate('common.gasFee')}</Row.Label>
-                <Row.Value>
-                  <Amount.Fiat value='10' />
-                </Row.Value>
-              </Row>
-              <Row fontSize='sm' fontWeight='medium'>
-                <Row.Label>{translate('common.fees')}</Row.Label>
-                <Row.Value>
-                  <Amount.Fiat value='0.0' />
-                </Row.Value>
-              </Row>
+              {
+                /* Only display gas fee if allowance has been granted - else we can't estimate them
+                 * Or maybe we just remove the fees row altogether because they're so cheap on L2?
```

Comment:
Actually not *that* cheap i.e 0.02$ ATM

<img width="407" alt="image" src="https://github.com/shapeshift/web/assets/17035424/2a95387d-0080-490f-8d95-b5a9c31aa924">

It's cheap, but not sub-penny cheap

Reply:
Also note the related asset shenanigans in this screenshot will get fixed in https://github.com/shapeshift/web/pull/6915 i.e since this is read only, there shouldn't be any related assets